readthedocs/notifications/messages.py

```diff
@@ -96,6 +96,19 @@ def get_rendered_body(self):
         ),
         type=ERROR,
     ),
+    Message(
+        id=BuildAppError.UPLOAD_FAILED,
+        header=_("There was a problem while updating your documentation"),
+        body=_(
+            textwrap.dedent(
+                """
+                Make sure your files are in the correct directory, or try again later.
```

Comment:
Just to be a little more explicit

```suggestion
                Make sure this project is outputting files to the correct directory, or try again later.
```